db/config.hh

```diff
@@ -61,6 +62,32 @@ struct seed_provider_type {
     }
 };
 
+/*
+ * This type is a glue between configuration in ms
+ * and duration
+ */
+struct duration_in_ms_type {
+    duration_in_ms_type() = default;
+    duration_in_ms_type(const duration_in_ms_type&) = default;
+    duration_in_ms_type(uint32_t n)
+            : _duration(n) {
+    }
+    duration_in_ms_type(const timeout_clock::duration &d);
+    duration_in_ms_type& operator=(const timeout_clock::duration &d);
+    uint32_t _duration = 0;
```

Comment:
We usually list data members on the top, not intermixed with member methods.

Reply:
I've forced-push a version that changes the order